source/reference/partition-based-sync.txt

```diff
@@ -6,6 +6,13 @@
 Partition-Based Sync
 ====================
 
+.. facet::
+   :name: genre 
+   :values: reference
+
+.. meta:: 
+   :description: Learn about the older sync mode for Atlas Device Sync, Partion-Based Sync.
```

Comment:
A bit nit-picky, but I don't think we want folks to "learn" about PBS anymore. What do you think about something like this?
```suggestion
   :description: Find information about the older sync mode for Atlas Device Sync, Partion-Based Sync.
```